tests/test_dataclass_reader.py

```diff
@@ -88,3 +89,29 @@ def test_csv_header_items_with_spaces_together_with_skipinitialspaces(create_csv
 
         assert user.name == 'User1'
         assert user.age == 40
+
+
+def test_parse_bool_value(create_csv):
```

Comment:
What you think about splitting this test in 3? One for true value, one for false and the last testing non-boolean values.

Reply:
Sounds good to me 👍